game-core/build.gradle

```diff
@@ -135,8 +135,9 @@ dependencies {
     }
 
     testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
-
-    testRuntime 'org.junit.jupiter:junit-jupiter-engine:5.0.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.1.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
+    testRuntime 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
     testRuntime 'org.junit.platform:junit-platform-launcher:1.0.2'
```

Comment:
Actually there's also a fourth that is currently needed as a workaround for eclipse (this is already fixed in the next major version that'll be released in June)
The version string should get updated to `1.1.0`

Reply:
Bumping this

Reply:
Updated in:  25feb59